File: exam_2019_july/PoolDay.java
import java.lang.Math;
import java.util.Scanner;

public class PoolDay 
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);

		int visitors = Integer.parseInt(sc.nextLine());
		double entranceFee = Double.parseDouble(sc.nextLine()); 
		double shezlongFee = Double.parseDouble(sc.nextLine());
		double umbrellaFee = Double.parseDouble(sc.nextLine());
		
		double totalFee = entranceFee * visitors + Math.ceil(visitors * 0.5) * umbrellaFee + Math.ceil(visitors * 0.75) * shezlongFee;
		
		System.out.printf("%.2f lv.", totalFee);
    }
}

File: exam_2019_july/PoolDay.cpp
#include <iostream>
#include <cmath>;

using namespace std;

int main()
{
	int visitors;
	cin >> visitors;
	
	double entranceFee;
	cin >> entranceFee;
	
	double shezlongFee;
	cin >> shezlongFee;
	
	double umbrellaFee;
	cin >> umbrellaFee;
	
	double totalFee = entranceFee * visitors + ceil(visitors * 0.5) * umbrellaFee + ceil(visitors * 0.75) * shezlongFee;
	
	cout.setf(ios::fixed);
	cout.precision(2);
	
	cout << totalFee << " lv." << endl;
	
	return 0;
}